receiver/mongodbatlasreceiver/README.md

```diff
@@ -13,27 +13,37 @@ as well as alerts via a configured [webhook](https://www.mongodb.com/docs/atlas/
 ## Getting Started
 
 The MongoDB Atlas receiver takes the following parameters. `public_key` and 
-`private_key` are the only two required values to receive metrics and logs and are obtained via the 
+`private_key` are the only two required values to receive metrics and logs and are obtained via the
 "API Keys" tab of the MongoDB Atlas Project Access Manager. In the example
 below both values are being pulled from the environment.
 
 In order to collect logs, at least one project must be specified. By default, logs for all clusters within a project will be collected. Clusters can be limited using either the `include_clusters` or `exclude_clusters` setting.
 
-MongoDB Atlas [Documentation](https://www.mongodb.com/docs/atlas/reference/api/logs/#logs) recommends a polling interval of 5 minutes. 
+MongoDB Atlas [Documentation](https://www.mongodb.com/docs/atlas/reference/api/logs/#logs) recommends a polling interval of 5 minutes.
 
-- `public_key` (required for metrics)
-- `private_key` (required for metrics)
+- `public_key` (required for metrics, logs, or alerts in `poll` mode)
+- `private_key` (required for metrics, logs, or alerts in `poll` mode)
 - `granularity` (default `PT1M` - See [MongoDB Atlas Documentation](https://docs.atlas.mongodb.com/reference/api/process-measurements/))
+- `storage` configure the component ID of a storage extension. If specified, alerts `poll` mode will utilize the extension to ensure alerts are not duplicated after a collector restart.
 - `retry_on_failure`
   - `enabled` (default true)
   - `initial_interval` (default 5s)
   - `max_interval` (default 30s)
   - `max_elapsed_time` (default 5m)
 - `alerts`
   - `enabled` (default false)
-  - `secret` (required if enabled)
-  - `endpoint` (required if enabled)
-  - `tls`
+  - `mode` (default `listen`. Options are `poll` or `listen`)
+  - `secret` (required if using `listen` mode)
+  - `endpoint` (required if using `listen` mode)
+  - `poll_interval` (default `5m`, only relevant using `poll` mode)
+  - `max_alert_processing` (default `50`)
+    - This is the number of alerts the receiver will try to process per poll. Alerts are ordered by creation time descending i.e. by default the 50 newewst ones processed will be processed but none further. Only relevant using `poll` mode.
```

Comment:
```suggestion
  - `max_alerts` (default `1000`)
    - When in `poll` mode, this is the maximum number of alerts the receiver will attempt to process per poll. Alerts are processed from most recently created to least recently created.
```

A few suggestions here. Most importantly, I think 50 is probably way too low for a default. 1000 should not take a long time to process and I would think is enough that most users will never have to worry about dropped alerts.